pages/fellows/maxymnaumchyk.md

```diff
@@ -20,6 +20,13 @@ mentors:
 
 proposal: /assets/pdf/fellows-2022/200-proposal-Maxym-Naumchyk.pdf
 presentations:
+  - title: Integrating ML algorithms for LHC data compression into the ESCAPE Virtual Research Environment
+    date: 2022-09-14
+    url: https://indico.cern.ch/event/1195270/contributions/5043782/attachments/2507754/4309461/Final%20project%20presentation%20Maxym%20Naumchyk.pdf
```

Comment:
```suggestion
    url: https://indico.cern.ch/event/1195270/contributions/5043782/
```

The URL of the attachment isn't stable in the case of any reupload change, so just use the contribution URL.